src/config.c

```diff
@@ -281,21 +283,35 @@ static RRStatus processConfigParam(const char *keyword, const char *value, Redis
         if (parseBool(value, &val) != RR_OK)
             goto invalid_value;
         target->tls_enabled = val;
+    } else if (!strcmp(keyword, CONF_TLS_TRACE)) {
```

Comment:
Is this leftover? I don't see where we use it.

Reply:
yes.